### ISSUE TITLE
generateChangelog does not recognize parameter includeObjects

### ISSUE THREAD

**Issue by evantill** [liquibase/liquibase#1147]
## Environment

**Liquibase Version**: version 3.8.9

**Liquibase Integration & Version**: CLI and gradle

**Liquibase Extension(s) & Version**: none (not sure)

**Database Vendor & Version**: all

**Operating System Type & Version**: macos 10.14.6

## Description

`generateChangelog` does not recognize parameter `includeObjects`.

According to this informations, `generateChangelog` should recognize parameter `includeObjects` :  
- [jira CORE-2847](https://liquibase.jira.com/browse/CORE-2847)
- [Liquibase forum](https://forum.liquibase.org/topic/using-includeobjects-excludeobjects-with-generatechangelog)

## Steps To Reproduce
Run `liquibase` from command line :  

```
liquibase /
--changeLogFile=xxx.xml /
--url=jdbc:oracle:thin:@xxx /
--username=xxx /
--password=xxx /
--diffTypes=data /
generateChangelog /
--includeObjects=XXX
```

## Actual Behavior
Error :  
```
12:48:54.731 INFO  [liquibase.integration.commandline.Main]: Starting Liquibase at mar., 12 mai 2020 12:48:54 CEST (version 3.8.9 #73 built at Mon Apr 06 09:32:58 UTC 2020)
12:48:55.289 INFO  [liquibase.integration.commandline.Main]: No Liquibase Pro license key supplied. Please set liquibaseProLicenseKey on command line or in liquibase.properties to use Liquibase Pro features.
12:48:55.290 INFO  [liquibase.integration.commandline.Main]: Liquibase Community 3.8.9 by Datical
Errors:
  Unexpected command parameter: '--includeObjects=XXX'
```

## Expected/Desired Behavior
Be able to use `includeObjects` with `generateChangelog`

**Comment by molivasdat** [comment]
Thanks @evantill for writing up this issue.  We will try to reproduce it using the repro steps you have documented and get back to you.

**Comment by molivasdat** [comment]
Hi @evantill .  It looks like --includeObjects has only ever been included as part of the ant task for generateChangeLogTask.  It sets that parameter as part of the DiffOutputControl object.

**Comment by evantill** [comment]
Will you consider adding it to `generateChangelog` ?

**Comment by molivasdat** [comment]
@evantill Yes we will take a look.  Will the other commands work in the meantime or do you have a PR to submit to add it?

**Comment by evantill** [comment]
no PR at the moment.

**Comment by ro-rah** [comment]
Hey @evantill , Not sure you are up for it but changing the code Liquibase code is pretty easy if you know Java.  Here's a [how-to](https://liquibase.jira.com/wiki/spaces/LB/pages/1274839216/How+to+Write+Code+for+Liquibase+Core) _if you are interested_.

And of course, I believe we already chat on Discord, there is a channel called #developers that you can get real time help with to do this feature or some other feature you think of.

Hope you are well!

Ronak
